*** server.xml.org	Sat Jun 14 23:03:54 2014
--- server.xml	Sat Jun 14 23:04:31 2014
***************
*** 69,75 ****
      -->
      <Connector port="8080" protocol="HTTP/1.1"
                 connectionTimeout="20000"
!                redirectPort="8443" />
      <!-- A "Connector" using the shared thread pool-->
      <!--
      <Connector executor="tomcatThreadPool"
--- 69,75 ----
      -->
      <Connector port="8080" protocol="HTTP/1.1"
                 connectionTimeout="20000"
!                redirectPort="8443" URIEncoding="UTF-8" />
      <!-- A "Connector" using the shared thread pool-->
      <!--
      <Connector executor="tomcatThreadPool"
***************
*** 124,129 ****
--- 124,135 ----
        <Host name="localhost"  appBase="webapps"
              unpackWARs="true" autoDeploy="true">

+ <Context path="/jira" docBase="${catalina.home}/atlassian-jira-6.1.war" reloadable="false" useHttpOnly="true">
+   <Resource name="UserTransaction" auth="Container" type="javax.transaction.UserTransaction"
+     factory="org.objectweb.jotm.UserTransactionFactory" jotm.timeout="60"/>
+   <Manager pathname=""/>
+ </Context>
+
          <!-- SingleSignOn valve, share authentication between web applications
               Documentation at: /docs/config/valve.html -->
          <!--
